src/SFA.DAS.EmploymentCheck.Functions/Application/Services/Learner/LearnerService.cs

```diff
@@ -49,6 +49,23 @@ IDataCollectionsResponseRepository repository
             _repository = repository;
         }
 
+        public async Task<IList<LearnerNiNumber>> GetDbNiNumbers(IList<Models.EmploymentCheck> employmentCheckBatch)
+        {
+            var learnerNiNumbers = new List<LearnerNiNumber>();
+            foreach (var employmentCheck in employmentCheckBatch)
+            {
+                LearnerNiNumber learnerNiNumber = null;
```

Comment:
Since we're only doing something with this variable on the happy path, you can remove this line and add `var` at the beginning of line 61 and move the `Add` to inside the if block

Reply:
Fixed.